validator_manager/src/remove_validator.rs

```diff
@@ -0,0 +1,240 @@
+use std::path::PathBuf;
+
+use types::PublicKeyBytes;
+use clap::{App, Arg, ArgMatches};
+use eth2::{lighthouse_vc::types::{DeleteKeystoreStatus, DeleteKeystoresRequest}, SensitiveUrl};
+use serde::{Deserialize, Serialize};
+
+use crate::{common::vc_http_client, DumpConfig};
+
+pub const CMD: &str = "remove";
```

Comment:
From my understanding, this is the wording that's shown on the CLI help text. I think it is better to have it named `delete` to be consistent with the Keymanager API: https://ethereum.github.io/keymanager-APIs/#/Local%20Key%20Manager

```suggestion
pub const CMD: &str = "delete";
```

Maybe can revise some of the wordings in this file, and also rename the name of this file to `delete_validators.rs`

Reply:
Fixed in [eadec07](https://github.com/chong-he/lighthouse/commit/eadec077e36d4e4742308bb139a919c8e5c4fe22)